protostar/testing/cheatcodes/expect_revert_cheatcode.py

```diff
@@ -18,6 +18,13 @@ def __init__(self):
 
     def expect_revert(self, expected_error: RevertableException) -> Callable[[], None]:
         if self._expected_error is not None:
+            if (
+                self._expected_error.error_type is None
+                and not self._expected_error.error_messages
+            ):
+                raise SimpleReportedException(
```

Comment:
CheatcodeException is more suitable here IMHO

Reply:
Done